metadata.rb

```diff
@@ -8,7 +8,7 @@
 
 recipe 'jenkins::master', 'Installs a Jenkins master'
 
-%w(ubuntu debian redhat centos scientific oracle amazon).each do |os|
+%w(ubuntu debian redhat centos scientific oracle amazon suse).each do |os|
```

Comment:
This should probably include opensuseleap since suse is just SLES

Reply:
We're using SLES specifically in our fork, I can totally modify this to add opensuse support if need be.

Reply:
adding test kitchen etc.